### PR TITLE
update signature logic + current addresses constants

### PR DESCRIPTION
**Pull request by livingrockrises**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: packages/modules/src/BatchedSessionRouterModule.ts [hunk 1]
```diff
@@ -150,21 +154,21 @@ export class BatchedSessionRouterModule extends BaseValidationModule {
       const proof = this.sessionKeyManagerModule.merkleTree.getHexProof(
         ethers.utils.keccak256(leafDataHex) as unknown as Buffer
       )
-      proofs.push(proof)
+
+      sessionDataTuple.push(proof)
+      sessionDataTuple.push('0x')
```
Comment (tomarsachin2271) [inline packages/modules/src/BatchedSessionRouterModule.ts:159]: This extra data should also come from sessionSignerData

File: packages/modules/src/BatchedSessionRouterModule.ts [hunk 2]
```diff
@@ -150,21 +154,21 @@ export class BatchedSessionRouterModule extends BaseValidationModule {
...
+
+      sessionDataTupleArray.push(sessionDataTuple)
     }
 
     // Generate the padded signature with (validUntil,validAfter,sessionVerificationModuleAddress,validationData,merkleProof,signature)
+
+    Logger.log('whole tuple ', sessionDataTupleArray)
```
Comment (tomarsachin2271) [inline packages/modules/src/BatchedSessionRouterModule.ts:166]: Should we change it to Logger.debug? Don't want to print any logs in normal mode.

File: packages/modules/src/utils/Constants.ts [hunk 3]
```diff
@@ -20,17 +20,17 @@ export const ECDSA_OWNERSHIP_MODULE_ADDRESSES_BY_VERSION = {
   V1_0_0: '0x0000001c5b32F37F5beA87BDD5374eB2aC54eA8e'
 }
 
-export const DEFAULT_SESSION_KEY_MANAGER_MODULE = '0x000002FbFfedd9B33F4E7156F2DE8D48945E7489'
+export const DEFAULT_SESSION_KEY_MANAGER_MODULE = '0x6E49e404BD70bcc4756F1057d2E2e6000cD38e1e'
```
Comment (tomarsachin2271) [inline packages/modules/src/utils/Constants.ts:23]: This address is not starting with initial 0s like other addresses?

File: packages/modules/src/BatchedSessionRouterModule.ts (hunk 2 above)
Comment (livingrockrises) [reply packages/modules/src/BatchedSessionRouterModule.ts:166]: ok will change it.

File: packages/modules/src/utils/Constants.ts (hunk 3 above)
Comment (livingrockrises) [reply packages/modules/src/utils/Constants.ts:23]: yes answered in our slack chat

File: packages/modules/src/BatchedSessionRouterModule.ts (hunk 1 above)
Comment (livingrockrises) [reply packages/modules/src/BatchedSessionRouterModule.ts:159]: you mean this '0x' ?
I will confirm with this @filmakarov if this is same as older additionalSig param used in session key manager module.

File: packages/account/package.json [hunk 4]
```diff
@@ -34,7 +34,7 @@
     "access": "public"
   },
   "dependencies": {
-    "@biconomy/account-contracts-v2": "npm:@biconomy/account-contracts-v2@^1.0.0"
+    "@biconomy-devx/account-contracts-v2": "npm:@biconomy-devx/account-contracts-v2@^1.0.0"
```
[the remark this answers is not in the document]
Comment (livingrockrises) [reply packages/account/package.json:37]: it's been devx only so far. check on npm there is no other package called account-contracts-v2

File: packages/modules/src/BatchedSessionRouterModule.ts (hunk 2 above)
Comment (livingrockrises) [reply packages/modules/src/BatchedSessionRouterModule.ts:166]: done

File: packages/modules/src/BatchedSessionRouterModule.ts (hunk 1 above)
Comment (tomarsachin2271) [reply packages/modules/src/BatchedSessionRouterModule.ts:159]: Yes i think it's the additionalSig param, in BatchedSessionRouter this param is passed to SVM.

Comment (livingrockrises) [reply packages/modules/src/BatchedSessionRouterModule.ts]: ok changed it to this
sessionDataTuple.push(sessionParam.additionalSessionData ?? '0x')

it was already part of types

**Review by tomarsachin2271 — APPROVED** [review]

**Pull request merged**